src/player.rs

```diff
@@ -0,0 +1,54 @@
+// use bevy_rapier2d::prelude::*;
```

Comment:
are we not using bevy_rapier2d at all at the moment?
if so not we can remove it from the Cargo.toml